app/api-v1/controllers/Recipe/helpers.js

```diff
@@ -1,10 +1,10 @@
 const mapRecipeData = (data) => ({
```

Comment:
sorry just realised we need another metadata key of `type: { type: 'LITERAL', value: 'RECIPE' }` which will help us later to identify tokens.

Also @mattdean-digicatapult I have a memory of us deciding to put something like `transactionId: { type: 'LITERAL', value: data.transaction.id }` to help the blockchain watcher match new tokens (that have no parents) to pending transactions

Reply:
mm... yes, this is something we do need for syncing up stuff. will add two props. To be fair this should not be blocking as we can make changes to token data on the fly and review would be deadly easy